README.md

```diff
@@ -140,43 +145,48 @@ directory (best to specify an absolute path though).
 
 You can also check if it has been built correctly with:
 
-    gst-inspect-1.0 builddir/gst-plugins/src/libgstdrpai.so
+```bash
+gst-inspect-1.0 builddir/gst-plugins/src/libgstdrpai.so
+```
 
 ## Some examples of running the plugin
 
 ### Read Camera and Show on Screen
 
-```
+```bash
 gst-launch-1.0 v4l2src device=/dev/video0 \
     ! videoconvert \
-    ! drpai model=yolov3 show-fps=true log-detects=true smooth-video-rate=30 \
+    ! drpai model=yolov3 log-detects=true smooth-video-rate=30 post-process-properties="show-fps=true" \
     ! videoconvert \
     ! autovideosink
 ```
+
 If your camera supports the BGR format (such as the coral camera), you can modify the camera size in 
 `~/v4l2init.sh` and skip the first `videoconvert` element like this:
-```
+
+```bash
 gst-launch-1.0 v4l2src device=/dev/video0 ! video/x-raw, width=640, height=480, format=BGR \
-    ! drpai model=yolov3 show-fps=true log-detects=true smooth-video-rate=30 \
+    ! drpai model=yolov3 log-detects=true smooth-video-rate=30 post-process-properties="show-fps=true" \
     ! videoconvert \
     ! autovideosink
 ```
+
 ### Read Camera and Stream on Network
 
 In case you already have the streaming working based on [here](StreamingVideo.md), you can 
 add the drpai element to the `stream.sh` file like this:
 
-````
+```bash
 #!/bin/bash
 [ $1 ] || { echo  "Please specify the destination IP address: ./stream.sh ip" >&2; exit 1; }
 
 ./v4l2-init.sh
 echo "Streaming to ${1} with DRPAI..."
 
 gst-launch-1.0 v4l2src device=/dev/video0 ! video/x-raw, width=640, height=480, format=BGR \
-    ! drpai model=yolov3 show-fps=true log-detects=true smooth-video-rate=30 \
+    ! drpai model=yolov3 log-detects=true smooth-video-rate=30 post-process-properties="show-fps=true" \
```

Comment:
This looks a little weird, is there a particular reason why it's not:
`post-process-properties="show-fps"` (which is `=true` implicitly, why do you specify it explicitly?

Reply:
Some items are enabled by default and need to be disabled, some others are disabled by default and need to be enabled. And some items are not boolean, they are integers or floats.

Reply:
Okay, fair enough, not sure if there's a better way to do this. It is acceptable, thanks for the small print!